### DISCUSSION TITLE
Add Notification for / in Comments Objects

### DISCUSSION THREAD

**Discussion by ZugunruheKami** [langfuse/langfuse#3336] [Ideas]
### Describe the feature or potential improvement

Thanks @marcklingen @marliessophie for the great initiative and work on Comments (https://github.com/orgs/langfuse/discussions/980)!

Currently when I leave comments, it doesn't notify the members in the team to view the comments. It will be useful to @ team members or to subscribe certain starred traces for a debug discussion. If subscribe to ceratin trace will be impractical but then subscribe to a specific UserId or TraceName with additional alerts when comments are added will  be interesting if one's actively monitoring and enhancing certain agent as his / her current task or when a specific UserId is tasked with testing. 

Of course easiest way to get notified may be the proactive searching to see what are the already commented objects which is the idea mentioned: https://github.com/orgs/langfuse/discussions/3335

One additional thought but not really urgent: our team also use Jira a lot hence current practice is to leave trace links in our jira tickets. Will be helpful to have a way to bind the two systems in an easy way.

**Comment by marliessophie** [comment]
Hey @Zugunruhekami, appreciate the feedback and glad you're enjoying comments! We've created a [issue](https://github.com/langfuse/langfuse/issues/3510) for tagging users in comments, which should track this aspect of your idea.

**Comment by enricchust** [comment]
@marliessophie  Hey, I really appreciate this initiative, it would be very helpful for me and my team, as we’re leaving a lot of comments on traces and prompts.

I completely agree that the first step should be enabling user mentions. Looking ahead, though, I’d love to suggest a broader vision: it would be great to have a dedicated section in the product, something like “Comments Visibility”, where we could view all comments made across the organization.

Ideally, this section would display a history of all comments, with links that take you directly to the relevant trace or prompt. This would be incredibly useful for teams managing AI products, especially when multiple engineers are contributing and leaving feedback, having a centralized view would make collaboration and oversight much easier.

I understand this could be quite a bit of work, I’m just leaving the idea here as something that would be genuinely valuable for our team to get a broader picture of everything that’s happening.

**Comment by jannikmaierhoefer** [comment]
Hi all, we added support for @mentions and reactions in comments here: https://langfuse.com/changelog/2025-11-04-comment-mentions-and-reactions

Let us know if you have feedback!
